### DISCUSSION TITLE
Temporary State (aka. client-side only state)

### DISCUSSION THREAD

**Discussion by tonysm** [livewire/livewire#3433] [3. Feature ideas]
I think [DOM Patching & Temporary Assigns from LV](https://hexdocs.pm/phoenix_live_view/dom-patching.html ) would be a good addition to LW. It looks like the `wire:ignore`, but instead of ignoring the tree, we could apply some smart patching to it and at the same time keep the state (`serverMemo`) small.

Has anyone attempted this? Or is there any security or practical implementations preventing this?

**Comment by joshhanley** [comment]
@tonysm there was a PR for partials by CSS selector, have a look here https://github.com/livewire/livewire/pull/2180

It was closed due to the CSS selector API, but if a better API can be found/used, I'm sure it'd be welcome.

I like the idea of temporary state.

If you get a chance, can you summarise the temporary assigns stuff here as well as some API ideas (or even just LiveView's syntax converted to Livewire), so others can understand what that might look like/mean for Livewire and how it could work?
